pyopentrustregion/testsuite.py

```diff
@@ -41,49 +41,49 @@
         raise FileNotFoundError("Cannot find testsuite library.")
 
 
-# define all tests
+# define all tests in alphabetical order
 fortran_tests = {
     "opentrustregion_tests": [
         "add_error_origin",
-        "truncated_conjugate_gradient",
-        "level_shifted_davidson",
-        "sanity_check",
-        "accept_trust_region_step",
-        "split_string_by_space",
-        "log",
-        "print_results",
-        "minres",
-        "jacobi_davidson_correction",
-        "orthogonal_projection",
         "abs_diag_precond",
-        "level_shifted_diag_precond",
-        "set_default",
-        "init_solver_settings",
-        "init_stability_settings",
-        "init_rng",
-        "gram_schmidt",
-        "generate_random_trial_vectors",
-        "generate_trial_vectors",
+        "accept_trust_region_step",
         "add_column",
+        "bisection",
+        "bracket",
         "extend_symm_matrix",
+        "generate_random_trial_vectors",
+        "generate_trial_vectors",
+        "gram_schmidt",
+        "init_rng",
+        "init_solver_settings",
+        "init_stability_settings",
+        "jacobi_davidson_correction",
+        "level_shifted_davidson",
+        "level_shifted_diag_precond",
+        "log",
         "min_eigval",
-        "symm_mat_min_eig",
-        "bracket",
+        "minres",
         "newton_step",
-        "bisection",
+        "orthogonal_projection",
+        "print_results",
+        "sanity_check",
+        "set_default",
         "solver",
+        "split_string_by_space",
         "stability_check",
+        "symm_mat_min_eig",
+        "truncated_conjugate_gradient",
```

Comment:
this is a lot of delta-LOC for something that is not relevant to this PR. It seems you've just alphabetized this list